org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/util/JRTUtil.java

```diff
@@ -160,9 +133,8 @@ public static JrtFileSystem getJrtSystem(File image, String release) throws IOEx
 					throw new RuntimeIOException(errorMessage, e);
 				}
 			});
-			return system;
```

Comment:
Please keep such code "as is" (here and below), it's otherwise hard to debug maps hits etc.

Reply:
Done, restored it.
In such cases I just look at the `<methodName> returned` field in the Variables view of the Debug perspective, but having it as a variable makes it even simpler.